cdxev/amend/command.py

```diff
@@ -2,7 +2,7 @@
 
 from cdxev.auxiliary.sbomFunctions import walk_components
 
-from .operations import Operation
+from .operations import Operation, ReplaceLicenseNameWithId
```

Comment:
It feels odd that you are using `ReplaceLicenseNameWithId` though you introduced the new function `process_license`